chains/evm/evmclient/evm-client.go

```diff
@@ -125,6 +124,18 @@ func (c *EVMClient) LatestBlock() (*big.Int, error) {
 	return head.Number, err
 }
 
+func (c *EVMClient) IsEIP1559Activated() (bool, error) {
+	if head, err := c.HeaderByNumber(context.TODO(), nil); err != nil {
+		if head.BaseFee != nil {
+			return true, nil
+		} else {
+			return false, nil
+		}
```

Comment:
Is this meant to be within the err block?

Reply:
No it isn't I will switch it